index.php

```diff
@@ -17,19 +21,45 @@
     'collections' => [
         'autoPublishedDrafts' => function ($site) {
             $autopublishfield = option("bvdputte.kirbyAutopublish.fieldName");
+            $autopublishfieldtoogle = option("bvdputte.kirbyAutopublish.fieldNameToogle");
             $drafts = $site->index()->drafts();
-            $autoPublishedDrafts = $drafts->filter(function ($draft) use ($autopublishfield) {
-                return ($draft->$autopublishfield()->exists()) && ($draft->$autopublishfield()->isNotEmpty()) && (empty($draft->errors()) === true);
+
+            return $drafts->filter(function ($draft) use ($autopublishfield, $autopublishfieldtoogle) {
```

Comment:
We could easily refactor this into a small function because the logic similar to the unpublish part. But we can leave it and call it a day ^^